src/es-module-shims.js

```diff
@@ -57,12 +57,11 @@ Object.defineProperties(importShim, {
   l: { value: undefined, writable: true },
   e: { value: undefined, writable: true }
 });
-importShim.fetch = url => fetch(url);
-importShim.skip = /^https?:\/\/(cdn\.pika\.dev|dev\.jspm\.io|jspm\.dev)\//;
+const importShimInitialOptions = self.importShimInitialOptions || {};
+importShim.fetch = importShimInitialOptions.fetch || (url => fetch(url));
+importShim.skip = importShimInitialOptions.skip || /^https?:\/\/(cdn\.pika\.dev|dev\.jspm\.io|jspm\.dev)\//;
+importShim.onerror = importShimInitialOptions.onerror || ((e) => { throw e; });
```

Comment:
In fact, we don't even need to expose these as instance-level hooks anymore, perhaps we make this a major change and treat `importShim` as fully stateless now.

Reply:
One reason I didn't remove the original API is this new API doesn't support changing the hook dynamically after the initial load by overwriting the hooks on the `importShim` instance. I wasn't sure if there might be valid use cases in the wild for the ability to swap out these hooks for subsequent module loads. 

If you're not aware of any use cases either, and don't mind removing support for it, I'd be more than happy to remove the old API. 🙂 

I usually get pretty nervous with these kinds of directly mutative APIs since they open up the possibility for some really nasty edge cases when users make mutations at inconvenient times. If we wanted to eventually re-introduce the ability to update these hooks after the initial load, I'd probably lean towards introducing some update function for users to call explicitly to update the hooks internally. That way we'd at least know exactly what's getting updated and when, so we can prepare the internals to accept the updates before actually applying them.